docs/concepts/aa-x-lit.md

```diff
@@ -0,0 +1,68 @@
+
+Section Overview
+
+- Use cases of account abstraction with Lit
+- An intro to account abstraction and multiparty computation (MPC)
+- How to use Lit Actions & Programmable Key Pairs (PKPs) to work with AA
+
+## How to use Lit with Account Abstraction (AA)
+
+1. Adding a Programmable Key Pair (PKP) as a signer to an AA wallet.
+2. User Onboarding - creating a smart contract account for someone new to web3. The signer can start as an Multiparty computation (MPC) key authorized through a web2 account.
+3. AA wallet authorization for a PKP - smart contract accounts with signing capabilities through PKPs.
+4. Non-ECDSA AA wallet with a PKP wallet - allowing freedom of signature verification scheme.
+5. Conditional gas payments - PKP wallet pays for gas fees when certain conditions are met.
+
+
+
+## Account Abstraction (AA) Intro
+
+Account abstraction references [EIP-4337](https://eip4337.com/en/latest/index.html?ref=spark.litprotocol.com). AA shifts the validation of transactions from the Ethereum protocol to the smart contract level with a specific [entry point](https://ethereum-magicians.org/t/erc-4337-account-abstraction-via-entry-point-contract-specification/7160/1?ref=spark.litprotocol.com). With it are abstractions for a user's account, standardized smart contract account interfaces, and gas abstraction. This is possible by separating the transaction's signature from the account address, allowing for possibilities like switching between different accounts in a single transaction. 
+
+EIP-4337 sets a standard interface for everyone to work with when creating smart contract accounts.
+
+How AA enhances user experience:
+
+- Programmed security - The requirement of additional confirmations in the event of fraud detection such as two-factor authentication, additional signing with a web3 wallet, or confirmation through another smart contract.
+- Social Recovery - In [What is a Social Recovery Wallet](https://hackernoon.com/what-is-a-social-recovery-wallet?ref=spark.litprotocol.com) by Vitalik Buterin, he writes a good wallet design needs to satisfy three key criteria: no single point of failure, low mental overhead, and maximum ease of transacting. Social recovery can look like a multi-signature transaction to approve changing a signing key if an account has been compromised or lost.
+
+## Multi party computation (MPC)
+
+MPC enables multiple parties – each holding private data – to evaluate a computation without ever revealing any of the private data held by each party (or any otherwise related secret information). 
+
+An MPC wallet is a smart contract wallet whose public private key is divided and shared among multiple parties, in Lit’s case the key is stored across the Lit nodes and only authorized parties can recombine the key shares to generate a signing key.
```

Comment:
"An MPC wallet is a smart contract wallet"

did we mean Lit MPC wallet?

Reply:
I think this section still holds, from Alchemy's [MPC article](https://www.alchemy.com/overviews/mpc-wallet) they describe an MPC wallet as, " an MPC wallet divides a single private key among multiple parties. "